inference-engine/ie_bridges/python/wheel/setup.py

```diff
@@ -210,6 +210,23 @@ def is_tool(name):
     return True
 
 
+def remove_rpath(file_path):
+    """
+        Remove rpath from binaries
+        :param file_path: binary path
+        :type file_path: pathlib.Path
+    """
+    if sys.platform == "darwin":
+        cmd = f'otool -l {file_path} ' \
+              f'| grep LC_RPATH -A 3 | grep -o "path.*" | cut -d \" \" -f2 ' \
+              f'| xargs -I{{}} install_name_tool -delete_rpath {{}} {file_path}'
+        ret_info = subprocess.run(cmd, shell=True, check=True)
```

Comment:
it is an insecure way to use shell=true, see https://bandit.readthedocs.io/en/latest/plugins/b602_subprocess_popen_with_shell_equals_true.html?highlight=%20shell%3DTrue#b602-subprocess-popen-with-shell-equals-true